### PR TITLE
LGA-2574 use new irsa service account

### PR DESCRIPTION
**Pull request by said-moj**
## What does this pull request do?

- Use new irsa service account instead of long lived acess key and secret key
- Remove long-lived aws environment variable settings
- Upgrade Celery to 4.0.2 to use Kombu version that uses botocore/boto3 for SQS communication
- Install celery[sqs] instead of just celery so that dependencies required to communicate with sqs are also installed

## Any other changes that would benefit highlighting?
Celery versions 4.0 and above require pycurl package and pycurl has some build dependencies that need to be installed. I added installing`curl-dev` to the built docker image and `libcurl4-gnutls-dev` to the pipeline.

Celery 4.0 does not automatically register class based tasks which means we have to manually register these tasks. More information https://docs.celeryq.dev/en/v4.0.0/whatsnew-4.0.html#the-task-base-class-no-longer-automatically-register-tasks and https://github.com/celery/celery/issues/5992

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_backend/libs/aws/s3.py [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+    def get_s3_connection(cls, bucket_name):
+        return S3Boto3Storage(bucket=bucket_name)
+
+    @classmethod
+    def download_file(cls, bucket_name, key):
+        try:
+            obj = cls.get_s3_connection(bucket_name).bucket.Object(cls.clean_name(key))
+            data = NamedTemporaryFile()
+            obj.download_fileobj(data)
+            data.seek(0)
```
Comment (jogilder) [inline cla_backend/libs/aws/s3.py:29]: can you put in a comment to explain why this line is here?

File: cla_backend/apps/reports/views.py [hunk 2]
```diff
@@ -201,18 +203,16 @@
...
+        obj = ReportsS3.download_file(bucket_name, key)
 
-        if k is None:
+        if obj is None:
             raise Http404("Export does not exist")
 
-        k.open_read()
-        headers = dict(k.resp.getheaders())
-        response = HttpResponse(k)
+        response = HttpResponse(obj["body"])
```
[the remark this answers is not in the document]
Comment (psweeting1) [reply cla_backend/apps/reports/views.py:213]: Should I worry about this?

[the remark this answers is not in the document]
Comment (said-moj) [reply cla_backend/apps/reports/views.py:213]: No it's a zip file that we push to S3 and downloading from S3 again

File: cla_backend/libs/aws/s3.py (hunk 1 above)
Comment (said-moj) [reply cla_backend/libs/aws/s3.py:29]: Done

**Review by jogilder — APPROVED** [review]
LGTM

**Pull request merged**